lib/ssf/client.rb

```diff
@@ -27,69 +27,21 @@ def send_to_socket(message)
       @socket.send(message, 0)
     end
 
-    def new_root_span(operation: '', tags: {})
+    def new_span(operation: '', tags: {}, trace_id: nil, parent_id: nil)
```

Comment:
I can change it to `start_span`. Re: passing in the parent, aren't there cases where you receive a `trace_id` and `parent_id` and want to start a span from a given context? (Like baskerville sending trace data to ticket-maker.)